test/benchmarks/benchmark_crc32.cc

```diff
@@ -58,6 +58,7 @@ class crc32: public benchmark::Fixture {
 
 BENCHMARK_CRC32(braid, PREFIX(crc32_braid), 1);
 BENCHMARK_CRC32(generic, PREFIX(crc32_c), 1);
+BENCHMARK_CRC32(chorba_sse, PREFIX(crc32_chorba_sse), 1);
```

Comment:
```c
#ifdef X86_CHORBA_SSE_CRC
```
Because the function is defined only if macro ``X86_CHORBA_SSE_CRC`` is defined.